.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+# For available hooks and configuration options see: https://pre-commit.com/hooks.html
+# To manage and run locally, first install `pre-commit` https://pre-commit.com/#installation
+# To auto update to the latest versions `pre-commit autoupdate`
+# To manually run against all files `pre-commit run --all-files`
+# To force hooks to run on local commits `pre-commit install`
+repos:
```

Comment:
@tyler-hoffman added usage explanation here

Reply:
Thanks. I think this is great, and your reasoning about not updating the readme seems fine.